.github/workflows/check.yml

```diff
@@ -34,6 +34,33 @@ jobs:
     uses: insightsengineering/r.pkg.template/.github/workflows/version-bump.yaml@main
     secrets:
       REPO_GITHUB_TOKEN: ${{ secrets.REPO_GITHUB_TOKEN }}
+  copy_vbump:
```

Comment:
@cicdguy can you please have a look?
It's a new need regardless of path parameter to workflows. The goal is to update DESC in `/book` (incl. vbump, date etc.) first in standard `vbump` workflow and then copy it all into the `/package/DESC`. The goal is to keep consistency between versioning of those two so as to avoid confusion.
We might also consider two vbumps in two subdirs separately. Maybe it's easier to maintain? 🤔

Reply:
Thinking about that a little bit more, we only need it on package level (so that it's present in CI that tlg.catalog.pkg@1.2.3.9004 failed). Having this on book level doesn't provide any value. Let me adjust that PR then

Reply:
I will just keep one in package not to confuse people. When within a book, one might want to add book-specific entries or (even worse) remove package-specific fields. Let's go with simpler at the beggining.